sgx/ias-types/src/report.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
```

Comment:
Fairly straightforward, simply the IAS report, as would be returned via HTTP. Without the validation code (which will come in a future PR), this structure is fairly straightforward.